lib/testScenario/gen/staticTestScenarioGenerator.ts

```diff
@@ -13,7 +13,7 @@ import { RawTestDefinitionFile, TestDefinitionFile, TestResources } from "./../t
 import { TestResourceLoaderOption } from "./../testResourceLoader";
 import { inversifyGetInstance, TYPES } from "./../../inversifyUtils";
 
-type GenerationRule = "put-delete" | "listOperation";
+type GenerationRule = "put-delete" | "operations-list";
```

Comment:
How about change `put-delete` to `operations-putDelete` to be consistent?

Reply:
emm. The put-delete is for resource put delete. But operations-list is for operationLists

Reply:
Let's rename it as `resource-putDelete`.